source/slang/slang-syntax.h

```diff
@@ -819,8 +819,10 @@ namespace Slang
             return result;
         }
 
-        Element* m_begin;
-        Element* m_end;
+        bool hasContent() const { return m_end != m_begin; }
```

Comment:
Okay, so `hasContent` *was* introduced here. I might argue for `isEmpty` as the more natural primitive, but I guess all the current call sites actually want the opposite. Would `isNonEmpty` be reasonable?